lib/tasks/sul.rake

```diff
@@ -470,7 +485,7 @@ namespace :sul do
     total_users = users.size
     puts "Number of active users with harvesting enabled: #{total_users}"
 
-    header_row = %w[orcidid name last_name first_name sunet cap_profile_id total_publications total_approved_publications total_new_publications
+    header_row = %w[orcidid name last_name first_name sunetid cap_profile_id total_publications total_approved_publications total_new_publications
                     total_denied_publications]
```

Comment:
we use `sunetid` as an existing import column name in the above rake task, just making this column consistent with the other export rake task